resources/views/partials/alerts/danger.blade.php

```diff
@@ -1,11 +1,6 @@
-<div class="row {{ $payload['classes'] }}" style="
-    padding: 15px;
-    color: #FFF;
-    background: #F86380;
-    border-radius: 5px;
-">
-    <div class="row__column row__column--compact">
-        <i class="fas fa-times"></i>
+<div class="row alert alert--danger {{ $payload['classes'] }}">
```

Comment:
I think since we have an `alert` class now, we can stop using `row` here and inherit the same logic that `row` has, into `alert`.